client/js/ui.js

```diff
@@ -1525,6 +1525,17 @@ function itemCard(id) {
   info_name.classList.add('inventory-info-name');
   info_name.innerText = item.name;
 
+  if (item.status) {
+    console.log(item.status, item.status_message);
```

Comment:
Is it ok to take this out before merging or will it help debug things later?

Reply:
Yeah, thanks for catching that. I'll take it out.